src/app/data/project-data.ts

```diff
@@ -392,6 +392,37 @@ export const projectsData = [
         ],
         tools: []
     },
+    {
+        name: "Animal Guide for ACNH",
+        description: "Fan-made guidebook for ACNH with comprehensive info on collectibles, villagers, and more",
+        languages: ["en"],
+        logo: "animal-guide-for-acnh.png",
+        logoStyle: "",
+        site: "https://animalguide.app/",
```

Comment:
We only list the site if it serves as a guide itself, since users filtering for websites won't be looking for mobile apps. If you want, you can set your Android and iOS links to animalguide.app.